### PR TITLE
fixed issue where the command tools wont be recognized if terminal/zs…

### PR DESCRIPTION
**Pull request by driff**
Fixes an issue where you would never get that the command tools are installed if your shell had additional messages, for example, mine was showing an error when shell started because it failed to read an env file.

### CODE REVIEW

File: PlayCover.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -762,7 +762,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				MACOSX_DEPLOYMENT_TARGET = 12.0;
+				MACOSX_DEPLOYMENT_TARGET = 13.1;
```
Comment (amirsaam) [inline PlayCover.xcodeproj/project.pbxproj]: Not acceptable.

File: PlayCover.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -783,14 +783,15 @@
 				ASSETCATALOG_COMPILER_INCLUDE_ALL_APPICON_ASSETS = NO;
 				BUNDLE_ID_SUFFIX = "";
 				CODE_SIGN_ENTITLEMENTS = PlayCover/PlayCoverRelease.entitlements;
-				CODE_SIGN_IDENTITY = "Developer ID Application";
-				CODE_SIGN_STYLE = Manual;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "Apple Development";
+				CODE_SIGN_STYLE = Automatic;
```
Comment (amirsaam) [inline PlayCover.xcodeproj/project.pbxproj]: You have pushed Code Signing changes

File: PlayCover.xcodeproj/project.pbxproj (hunk 1 above)
Comment (driff) [reply PlayCover.xcodeproj/project.pbxproj]: fixed

File: PlayCover.xcodeproj/project.pbxproj (hunk 2 above)
Comment (driff) [reply PlayCover.xcodeproj/project.pbxproj]: fixed

File: PlayCover/Utils/Shell.swift [hunk 3]
```diff
@@ -75,18 +75,17 @@ class Shell: ObservableObject {
...
 
 	static let isXcodeCliToolsInstalled: Bool = {
-        let toolsPath = try? sh("xcode-select -p")
-        if let toolsPath = toolsPath?.trimmingCharacters(in: .whitespacesAndNewlines) {
-            if FileManager.default.fileExists(atPath:
-                toolsPath.appending("/usr/bin/notarytool")) { // This executable is gaurenteed
-                                                              // to exist weather it's the CLT or the full xcode
-                return true
-            }
+        let trimmedPath = getPath(path: "xcode-select -p")?.split(separator: "\n").last?.trimmingCharacters(in: .whitespacesAndNewlines)
```
Comment (amirsaam) [inline PlayCover/Utils/Shell.swift]: You need to break lines here to avoid SwiftLint error.

**Comment by IsaacMarovitz** [comment]
Will no longer be needed due to #711

**Review by Depal1 — APPROVED** [review]

**Comment by Depal1** [comment]
This will be merged as part of a 2.0.3 bug fix release.

**Pull request merged**